workers/data_refinery_workers/processors/utils.py

```diff
@@ -66,6 +71,7 @@ def start_job(job_context: Dict):
 
         # Just in case
         job_context["original_files"] = []
+        job_context["computed_files"] = []
```

Comment:
It seems like this line could be outside of the `if` instead of both of its branches.